server.go

```diff
@@ -64,7 +64,12 @@ type Server struct {
 	TraceAddr   *net.UDPAddr
 	RcvbufBytes int
 
+	Interval             time.Duration
```

Comment:
That's a good point. I regularly forget the upper/lower distinct. Very few of these need to be public, so will adjust!